src/fd.h

```diff
@@ -11,34 +11,6 @@
 #ifndef SQUID_FD_H_
 #define SQUID_FD_H_
 
-namespace Comm {
-
-/// An open Comm-registered file descriptor guard that, upon creation, registers
-/// the descriptor with Comm and, upon destruction, unregisters and closes the
-/// descriptor (unless the descriptor has been release()d by then).
-class Descriptor
```

Comment:
I removed Comm::Descriptor as no longer used. AFAIK, there are no known problems with that class, and the long-term plan of using it for controlling descriptors lifetime is still valid, but we should not keep unused code in Squid. We can always re-add it later.